src/commands/trigger-workflows-for-all-modified.yml

```diff
@@ -269,6 +269,7 @@ steps:
           workflow_id=$(jq -r .id /tmp/swissknife/triggered_workflow.json)
           echo ""
           echo "Waiting for workflow: ${workflow_id} to complete"
+          sleep << parameters.sleep-time-between-checks >>
```

Comment:
Is this b/c it takes time for a triggered workflow to return a proper response in the API below ? we're sleeping here before our first try, which is potentially unnecessary

Reply:
indeed... 
Maybe we can add another if to skip the initial sleep
```sh
if (( $total == 0 )); then
  sleep << parameters.sleep-time-between-checks >>
fi
```

Reply:
Done